README.md

```diff
@@ -232,6 +205,5 @@ git push --tags
 
 ## Credits
 
-This is a heavily refactored version of 
-[basrijn's Outback_Mate3 library](https://github.com/basrijn/Outback_Mate3).
-Thank you basrijn!
+This was originally a heavily refactored version of
+[basrijn's Outback_Mate3 library](https://github.com/basrijn/Outback_Mate3), though has largely been completely rewritten since. Thanks anyway basrijn!
```

Comment:
Just FYI – I'll have a read through this once-merged (and therefore easier to read) and may make some tweaks